amethyst_animation/src/util.rs

```diff
@@ -20,11 +20,7 @@ where
     I: Send + Sync + 'static,
     T: AnimationSampling,
 {
-    // TODO: replace with entry API when specs is released
-    if let None = controls.get(entity) {
-        controls.insert(entity, AnimationControlSet::default());
-    }
-    controls.get_mut(entity).unwrap()
```

Comment:
@Rhuagh There's a tiny bug here.  This will panic if you call it with a dead entity.  I've preserved the bug as I don't want this PR to have a controversial change in it, but this function could panic.

Reply:
Ah, good call. Please file an issue for it so I don't forget.

Reply:
Should be using Entry API anyway.

Reply:
I've updated it to do that, but the problem is I can't make this behave correctly without altering the return type of the function, and I'm declaring that to be out of scope for this PR.

Reply:
I guess we need to return a result or option here ?